tests/system_tests/test_blueapi_system.py

```diff
@@ -66,7 +66,7 @@ def test_get_plans_by_name(client: BlueapiClient, expected_plans: PlanResponse):
 def test_get_non_existent_plan(client: BlueapiClient):
     with pytest.raises(KeyError) as exception:
         client.get_plan("Not exists")
-        assert str(exception) == ("{'detail': 'Item not found'}")
+    assert str(exception) == ("{'detail': 'Item not found'}")
```

Comment:
Could all these use `raises(match='pattern')` instead of having a separate assert? Something like

    with pytest.raises(KeyError, match=NOT_FOUND):
        client.get_plan("Not exists")

Reply:
Yep, think this is done in #655- after that and #619 I'll rebase this and see what still needs to be done